shared/js/participantStore.js

```diff
@@ -41,14 +41,14 @@ loop.store.ParticipantStore = function() {
 
     updatedParticipant(actionData) {
       this._updateParticipantData(actionData.userId, {
-        name: actionData.participantName
+        participantName: actionData.participantName
       });
     },
 
     updatedPresence(actionData) {
       this._updateParticipantData(actionData.userId, {
         isHere: actionData.isHere,
-        localPingTime: Date.now() - actionData.pingedAgo
+        localPingTime: actionData.pingedAgo
```

Comment:
@Mardak I think the timestaps calculated in the store are wrong. Let's imagine I've updated my ping with Date.now() - 1000 so the localPingTime=1000 then I invoke getOnlineParticipants and Date.now() - participant.localPingTime <= 60000 is false because of that, I'm not online.